ovc5/software/libovc/include/libovc/ethernet_packetdef.hpp

```diff
@@ -26,13 +26,14 @@ typedef struct __attribute__((__packed__))
 {
   uint64_t t_sec;
   uint64_t t_nsec;
+  uint8_t camera_id;
```

Comment:
Since the struct is packed we should try to keep everything as aligned to 64 bit words as possible (i.e. move the `camera_id` definition after `frame_id`), or move all the other unaligned fields such as bit_depth, height above.